packages/backend/src/tokens/jwt/signJwt.test.ts

```diff
@@ -0,0 +1,36 @@
+import type QUnit from 'qunit';
+
+import {
+  mockJwtHeader,
+  mockJwtPayload,
+  pemEncodedPublicKey,
+  pemEncodedSignKey,
+  publicJwks,
+  signingJwks,
+} from '../fixtures';
+import { __unstable__signJwt } from './signJwt';
+import { decodeJwt, hasValidSignature } from './verifyJwt';
+
+export default (QUnit: QUnit) => {
+  const { module, test } = QUnit;
+
+  module('signJwt(payload, options)', () => {
+    test('signs a JWT with a JWK formatted secret', async assert => {
+      const jwt = await __unstable__signJwt(mockJwtPayload, signingJwks, {
```

Comment:
❓ Is the generated signature static? Perhaps we can add an assertion for signature (string) equality if so. It makes sense to assert symmetric behavior with `verifyJwt` or `hasValidSignature` but it may also cause a edge case of "two wrongs making one right".

Reply:
No unfortunately the jwt is not static. 

Each time we sign the JWT it gets an updated IAT timestamp that changes the payload and the signature. 

The `hasValidSignature` test has its own check though with a pregenerated JWT for this reason.